examples/flow/stokesNonNewtonian.ipynb

```diff
@@ -364,7 +364,7 @@
     "                   xmin=0., xmax=1., ymin=0., ymax=1.)\n",
     "    except:\n",
     "        viewer = Viewer(vars=(velocity),\n",
-    "                   xmin=0., xmax=1., ymin=0., ymax=1., colorbar=True,scale=5)        "
+    "                   xmin=0., xmax=1., ymin=0., ymax=1., colorbar='vertical',scale=5)        "
```

Comment:
I don't think I'm actually running this as it's after the except. This notebook isn't part of the test suite yet.

Reply:
Ok, I ran it by raising an exception in the try block. If I do,
```
if __name__ == '__main__':
    try:
        raise RuntimeError('hello')
        from fipy.viewers.matplotlibViewer import MatplotlibStreamViewer
        viewer = MatplotlibStreamViewer(vars=(velocity),
                   xmin=0., xmax=1., ymin=0., ymax=1.)
    except:
        viewer = Viewer(vars=(velocity),
                   xmin=0., xmax=1., ymin=0., ymax=1., colorbar='vertical',scale=5)        
```
then I get

![index](https://user-images.githubusercontent.com/1986844/142499813-c7d06d51-78ad-45fb-a84e-15a370a59b6a.png)

which doesn't actually have a colorbar, but it doesn't need one either.

Reply:
Looks like the colorbar got copied over from stokesCavity.py (which displays several variables).

Reply:
`colorbar=` removed